android-project/app/src/main/java/org/libsdl/app/SDLActivity.java

```diff
@@ -89,7 +89,7 @@ public static void debugSource(int sources, String prefix) {
                 | InputDevice.SOURCE_CLASS_POSITION
                 | InputDevice.SOURCE_CLASS_TRACKBALL);
 
-        if (s2 != 0) cls += "Some_Unknown";
+        if (s2 != 0) cls += "Some_Unkown";
```

Comment:
Why did they decide to miss spell this ... maybe qndel fixed it, just not upstream.

Reply:
https://github.com/libsdl-org/SDL/pull/10827